### PR TITLE
CORE-12587. [SDK] Move SYSTEM_POWER_INFORMATION to ntpoapi.h

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

As Thomas Faber answered.

JIRA issue: [CORE-12587](https://jira.reactos.org/browse/CORE-12587)

### CODE REVIEW

File: modules/rostests/apitests/powrprof/power.c [hunk 1]
```diff
@@ -14,6 +14,7 @@
 #include <ntstatus.h>
 #define WIN32_NO_STATUS
 #include <winreg.h>
+#include <potypes.h>
```
Comment (SergeGautherie) [inline modules/rostests/apitests/powrprof/power.c]: > SYSTEM_POWER_INFORMATION does belong in ntpoapi only, which means if you need it in user mode, it must go in ndk/potypes.h enclosed in a NTOS_MODE_USER ifdef. 

@ThFabba, should I even simply copy the definition here, as I assume it should be needed in this apitest only?

Comment (ThFabba) [reply modules/rostests/apitests/powrprof/power.c]: No, NDK is the right place.
Can you include this as `<ndk/potypes.h>` please though?

File: sdk/include/ndk/potypes.h [hunk 2]
```diff
@@ -25,6 +25,13 @@ Header Name:
 #include <umtypes.h>
 #ifndef NTOS_MODE_USER
 #include <ntpoapi.h>
+#else
+typedef struct _SYSTEM_POWER_INFORMATION {
```
Comment (ThFabba) [inline sdk/include/ndk/potypes.h]: Please put this in its own ifdef, not an else, and give it a heading like:
```
//
// Information Structures for NtPowerInformation
//
```
Also, the brace goes on its own line

File: sdk/include/ddk/ntpoapi.h [hunk 3]
```diff
@@ -30,6 +30,13 @@ extern "C" {
 #ifndef _PO_DDK_
 #define _PO_DDK_
 
+typedef struct _SYSTEM_POWER_INFORMATION {
```
Comment (ThFabba) [inline sdk/include/ddk/ntpoapi.h]: Put this somewhere around PROCESSOR_STATE_HANDLER2 in L265/272 please.

File: modules/rostests/apitests/powrprof/power.c (hunk 1 above)
Comment (SergeGautherie) [reply modules/rostests/apitests/powrprof/power.c]: Done.

File: sdk/include/ddk/ntpoapi.h (hunk 3 above)
Comment (SergeGautherie) [reply sdk/include/ddk/ntpoapi.h]: Done.

File: sdk/include/ndk/potypes.h (hunk 2 above)
Comment (SergeGautherie) [reply sdk/include/ndk/potypes.h]: Done.

**Pull request merged**